src/core/config/ip6.h

```diff
@@ -177,4 +177,12 @@
 #define OPENTHREAD_CONFIG_UNSECURE_TRAFFIC_MANAGED_BY_STACK_ENABLE 0
 #endif
 
+/**
+ *
+ * This setting configures the Multicast Listener Registration parent proxing in Thread 1.2.
+ * This is compulsory for 1.2 FTD.
+ *
+ */
+#define OPENTHREAD_CONFIG_MLR_PROXY_ENABLE (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2) && OPENTHREAD_FTD
```

Comment:
Maybe more proper to put in src/core/config/tmf.h? as it mainly related to the CoAP transaction.

Reply:
moved